src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,115 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationService REGISTRATION_SERVICE = new RegistrationServiceImpl();
+
+    @Test
+    void register_checkOnUserRegistration_isOk() {
+        User actual = new User();
+        actual.setLogin("Snowman");
+        actual.setPassword("123456");
+        actual.setAge(44);
+        User expected = REGISTRATION_SERVICE.register(actual);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_theSameLogin_NotOk() {
+        User testUser = new User();
+        testUser.setLogin("WormJim");
+        testUser.setPassword("123456");
+        testUser.setAge(29);
+        REGISTRATION_SERVICE.register(testUser);
+        User testUser2 = new User();
+        testUser2.setLogin("WormJim");
+        testUser2.setPassword("654321");
+        testUser2.setAge(24);
+        assertThrows(ValidationException.class, () -> {
+            REGISTRATION_SERVICE.register(testUser2);
+        });
+    }
```

Comment:
In the test case `register_theSameLogin_NotOk`, you are using the `REGISTRATION_SERVICE.register()` method to add a user to the storage. This is not a good practice, because you are testing the same method that you use to set up the test. Consider using a low-level method to add a user to the storage, such as `Storage.people.add()`.